### PR TITLE
Split longer insight into smaller ones (5)

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/blockchain/fundamentals/building-the-ledger/verifying-the-ledger.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/blockchain/fundamentals/building-the-ledger/verifying-the-ledger.md?ref=58f84d70af1dd8bddfbb7547340a5611542c2d01)

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - blockchain/fundamentals/building-the-ledger/verifying-the-ledger-iii.md

**Comment by enki-hq** [comment]
Rule insight:questions failed for files: 
  - blockchain/fundamentals/building-the-ledger/verifying-the-ledger-iii.md

File: blockchain/fundamentals/building-the-ledger/verifying-the-ledger-ii.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+  - introduction
+
+  - workout
+
+links:
+  - '[Full list of block checks](https://en.bitcoin.it/wiki/Protocol_rules#.22block.22_messages)'
+
+---
+# Transaction verification and the double-spending problem
```
Comment (kapnobatai137) [inline blockchain/fundamentals/building-the-ledger/verifying-the-ledger-ii.md]:
```suggestion
# Transaction Verification and the Double-Spending Problem
```

File: blockchain/fundamentals/building-the-ledger/verifying-the-ledger-iii.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+  - introduction
+
+  - workout
+
+links:
+  - '[Full list of block checks](https://en.bitcoin.it/wiki/Protocol_rules#.22block.22_messages)'
+
+---
+# Block verification
```
Comment (kapnobatai137) [inline blockchain/fundamentals/building-the-ledger/verifying-the-ledger-iii.md]:
```suggestion
# Block Verification
```

**Pull request merged**